apps/trustlab/src/payload/blocks/Banner.js

```diff
@@ -0,0 +1,50 @@
+import { validateHexColor } from "@commons-ui/payload";
+
+const Banner = {
+  slug: "banner",
+  imageURL: "/images/cms/blocks/banner.png",
+  fields: [
+    {
+      type: "text",
+      name: "title",
+      required: true,
+    },
+    {
+      type: "textarea",
+      name: "description",
+    },
+    {
+      type: "upload",
+      name: "image",
+      relationTo: "media",
+    },
+    {
+      type: "row",
+      fields: [
+        {
+          type: "text",
+          name: "backgroundColor",
+          defaultValue: "#02041C",
```

Comment:
Lets use/create a proper color field. Quick googlin' shows:

- https://www.npmjs.com/package/@nouance/payload-better-fields-plugin
- https://www.npmjs.com/package/@modernaut/payload-color-swatch-field

Lets review the packages (popularity, history, etc.) and pick one. Otherwise, we can also [implement](https://oleksii-s.dev/blog/how-to-build-color-input-in-payload-cms-3) a custom one.